mockwebserver/pom.xml

```diff
@@ -14,18 +14,18 @@
 
   <dependencies>
     <dependency>
-      <groupId>com.squareup.okhttp</groupId>
+      <groupId>com.squareup.okhttp3</groupId>
```

Comment:
If you care to, all these dependencies on sibling artifacts can use `${project.groupId}`.